pkgs/applications/misc/collision/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, callPackage
+, crystal
+, libadwaita
+, gtk3
```

Comment:
Collision doesn't use GTK 3

Reply:
Package `gtk3` provide `gtk-update-icon-cache` command needed by `make install`
```
> make: gtk-update-icon-cache: No such file or directory
```

Reply:
I should probably fix this upstream but gtk4 provides `gtk4-update-icon-cache`

Reply:
Or i can just patch this :D It works

Reply:
I think the fewer dependencies in the package is better